modules/unauthenticated-users-cluster-role-binding.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * authentication/impersonating-system-admin.adoc
+// * cicd/builds/triggering-builds-build-hooks.adoc
+// * authentication/tokens-scoping.adoc
+// * authentication/managing-oauth-access-tokens.adoc
+// * post_installation_configuration/preparing-for-users.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="unauthenticated-users-cluster-role-bindings_{context}"]
+= Adding unauthenticated groups to cluster role bindings
+
+As a cluster administrator, you can add unauthenticated users to the following cluster role bindings in {product-title}. Unauthenticated users do not have access to most cluster role bindings by default.
```

Comment:
```suggestion
As a cluster administrator, you can add unauthenticated users to the following cluster roles in {product-title} by creating a (cluster-)role-binding. Unauthenticated users do not have access to non-public cluster roles.
```

Reply:
Can I say "private cluster roles" instead of "non-public?" Or is there a difference between the two that is relevant here?

Reply:
I think the emphasis is on "public" as it might come as a surprise to everyone that there are actually 2 public cluster roles. They have "public" in their names as well... If we only say "private cluster roles", the immediate question that arises is: "are not all of them private"?